agents_json/devopness/agents.json

```diff
@@ -0,0 +1,206 @@
+            "actionId": "devopness_list_project_environments",
+            "fieldPath": "responses.200.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "responses.200.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "parameters.credential_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_create_server",
+            "fieldPath": "requestBody.credential_id"
+          }
+        }
+      ],
+      "fields": {
+        "parameters": [
+          {
+            "name": "hostname",
+            "type": "string",
+            "required": true,
+            "description": "The server name and hostname to be set on Linux cloud servers. Accepts numbers (0-9), dash (-) and lower case non accented characters. Must not be greater than 60 characters."
+          },
+          {
+            "name": "provision_input",
+            "required": true,
+            "type": "object",
+            "description": "Configuration used to launch the new server as a virtual machine instance in the cloud provider",
+            "properties": {
+              "credential_id": {
+                "type": "integer",
+                "description": "The unique id of the cloud provider credential to be used on connect to cloud provider",
+                "example": 8430015757
+              },
+              "cloud_service_code": {
+                "$ref": "#/components/schemas/ServerCloudServiceCode"
+              },
+              "settings": {
+                "description": "The configurations used to cloud service",
+                "anyOf": [
+                  {
+                    "$ref": "#/components/schemas/CloudServiceSettingsAwsEc2"
+                  },
+                  {
+                    "$ref": "#/components/schemas/CloudServiceSettingsDigitalOceanDroplet"
+                  },
+                  {
+                    "$ref": "#/components/schemas/CloudServiceSettingsGcpGce"
+                  },
+                  {
+                    "$ref": "#/components/schemas/CloudServiceSettingsSelfHostedCustom"
+                  }
+                ]
+              }
+            }
+          }
+        ],
+        "requestBody": {},
+        "responses": {
```

Comment:
@yagnyaPatel 
Do I need to define/copy the `responses` field?
Or when `responses` element is not present the ones defined in the OpenAPI spec are going to be used by default?

Reply:
Updated to use `response.success`, copying response object properties and example.
Does it look good good now?